mobile-widgets/qmlmanager.cpp

```diff
@@ -108,6 +109,24 @@ extern "C" int gitProgressCB(const char *text)
 	return 0;
 }
 
+// this callback is used from the uiNotification() function
+// the detour via callback allows us to keep the core code independent from QMLManager
+// I'm not sure it makes sense to have three different progress callbacks,
+// but the usage models (and the situations in the program flow where they are used)
+// are really vastly different...
+// this is mainly intended for the early stages of the app so the user sees that
+// things are progressing
+extern void showProgress(QString msg)
```

Comment:
This should read `static`, no?

Reply:
I need to look at that again... I think this is yet another left over from my initial idea of what I wanted to do there.